.goreleaser.yaml

```
@@ -0,0 +1,64 @@
+builds:
+# clusterctl-aws
+- id: "clusterctl-aws"
+  main: ./cmd/clusterawsadm
+  binary: bin/clusterctl-aws
+  env:
+    - CGO_ENABLED=0
+  ldflags:
+  - -s -w 
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitMajor={{.Major}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitMinor={{.Minor}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitVersion={{.Version}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitCommit={{.Commit}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitTreeState={{.GitTreeState}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.buildDate={{.Date}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/cmd/clusterawsadm/cmd/version.CLIName=clusterctl-aws'
+  goos:
+  - linux
+  - darwin
+  - windows 
+  goarch:
+  - amd64
+  - arm64
+
+# clusterawsadm
+- id: "clusterawsadm"
+  main: ./cmd/clusterawsadm
+  binary: bin/clusterawsadm
+  env:
+    - CGO_ENABLED=0
+  ldflags:
+  - -s -w 
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitMajor={{.Major}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitMinor={{.Minor}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitVersion={{.Version}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitCommit={{.Commit}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.gitTreeState={{.GitTreeState}}'
+  - -X 'sigs.k8s.io/cluster-api-provider-aws/v2/version.buildDate={{.Date}}'
+  goos:
+  - linux
+  - darwin
+  - windows 
+  goarch:
+  - amd64
+  - arm64
+
+archives:
+- id: clusterctl-aws
+  builds:
+  - clusterctl-aws
+  format_overrides:
+  - goos: windows
+    format: zip
+  name_template: "clusterctl-aws_{{ .Tag }}_{{ .Os }}_{{ .Arch }}"
+- id: clusterawsadm
+  builds:
+  - clusterawsadm
+  format_overrides:
+  - goos: windows
+    format: zip
+  name_template: "clusterawsadm_{{ .Tag }}_{{ .Os }}_{{ .Arch }}"
+
+changelog:
+  disable: true
```

```suggestion
  disable: true

release:
  discussion_category_name: General

  extra_files:
    - glob: ./templates/*.yaml

  draft: true
```

This suggestion comes from examining the contents of https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases/tag/v2.4.1. Mostly, it's including the template files which are fetched by clusterctl.

It also moves the choice to be a draft into the configuration file vs a command line argument.